namespaces/live.cloud-platform.service.justice.gov.uk/offender-events-dev/resources/pic-probation-offender-events-sub-queue.tf

```diff
@@ -99,10 +99,10 @@ resource "kubernetes_secret" "pic_probation_offender_events_dead_letter_queue" {
   }
 }
 
-resource "kubernetes_secret" "pic_probation_sqs_tool_main_queue" {
+resource "kubernetes_secret" "pic_probation_offender_events_sqs_tool_main_queue" {
```

Comment:
I still don't see what the relevance of the `sqs-tool` bit is? I'm guessing this is copy-pasted but if it has no meaning for us then we shouldn't keep it just because it was there before `pic_probation_offender_events_main_queue` seems fine to me unless there's some specific reason why we need `sqs-tool`.

Reply:
Sorry to be picky with these ones but it's the sort of thing that we will change once and never look at again until something explodes in 18 months time! 😄 At which point we don't want to be wondering what this SQS tool is supposed to be and if it's relevant to our fix.

Reply:
Actually on further thought - we should have the name of the service in there (which coming to think of it I think is what the `sqs-tool` bit is about) so that if another service starts receiving offender events there's no clash. See suggestion below.